crates/bitwarden-wasm-internal/src/pure_crypto.rs

```diff
@@ -323,6 +323,20 @@ impl PureCrypto {
         #[allow(deprecated)]
         dangerous_derive_kdf_material(password, salt, &kdf)
     }
+
+    pub fn decrypt_user_key_with_master_key(
+        encrypted_user_key: String,
+        master_key: Vec<u8>,
+    ) -> Result<Vec<u8>, CryptoError> {
+        let master_key = &BitwardenLegacyKeyBytes::from(master_key);
+        let master_key = &SymmetricCryptoKey::try_from(master_key)?;
+        let master_key = MasterKey::try_from(master_key).map_err(|_| CryptoError::InvalidKey)?;
```

Comment:
The try_from impl is already returning invalidkey, so no need to map it over again
```suggestion
        let master_key = MasterKey::try_from(master_key)?;
```

Reply:
Fixed in 59df3cb6